### PR TITLE
Tags csv export front

### PR DESCRIPTION
**Pull request by cabch**
+ also changed ocpp local params export

### CODE REVIEW

File: src/app/pages/users/tags/tags-list-table-data-source.ts [hunk 1]
```diff
@@ -314,6 +318,15 @@ export class TagsListTableDataSource extends TableDataSource<Tag> {
           (actionDef as TableImportTagsActionDef).action(ImportDialogComponent, this.dialog);
         }
         break;
+      case UserButtonAction.EXPORT_TAGS:
+        if (actionDef.action) {
+          const filterValues = this.buildFilterValues();
+          filterValues['WithUser'] = 'true';
```
Comment (LucasBrazi06) [inline src/app/pages/users/tags/tags-list-table-data-source.ts:324]: I don't think this filtering has to be done here.
Filtering of the list should be reused.
If extra params need to be forced then add them in the backend.

File: src/assets/i18n/de.json [hunk 2]
```diff
@@ -1299,7 +1299,10 @@
         "description": "Dieser Nutzer wurde nicht mit dem Rechnungssystem synchronisiert.",
         "action": "Nutzer synchronisieren."
       }
-    }
+    },
+    "export_tags_title": "Export Tags",
+    "export_tags_confirm": "Do you really want to export the Tag list to a CSV format?",
+    "export_tags_error": "Error occurred while trying to export the Tags"
```
Comment (LucasBrazi06) [inline src/assets/i18n/de.json:1305]: "Error occurred while trying to export the Tags, check the logs"

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]
Small changes

File: src/app/pages/users/tags/tags-list-table-data-source.ts (hunk 1 above)
Comment (cabch) [reply src/app/pages/users/tags/tags-list-table-data-source.ts:324]: I forgot to ask, will we let the choice to the user (checkbox or whatever) to chose if he wants to export with users or not ?

Comment (LucasBrazi06) [reply src/app/pages/users/tags/tags-list-table-data-source.ts:324]: For the time being let's always export all the tags with all the users and we'll see if the customers request this feature.

**Pull request merged**